GiniVision/Classes/Core/Multipage Review/MultipageReviewViewController.swift

```diff
@@ -392,9 +395,9 @@ extension MultipageReviewViewController {
     }
     
     fileprivate func createReorderPagesTip() {
-        blurEffect = UIVisualEffectView(effect: UIBlurEffect(style: .light))
-        blurEffect?.alpha = 0
-        self.view.addSubview(blurEffect!)
+        opaqueView = OpaqueViewFactory.create(with: giniConfiguration.multipageToolTipOpaqueBackgroundStyle)
+        opaqueView?.alpha = 0
+        self.view.addSubview(opaqueView!)
```

Comment:
You can avoid this force unwrapping if you work with a local variable, set it up and at the end of the method assign `opaqueView` to it. But not super important.